packages/wrangler/src/queues/cli/commands/create.ts

```diff
@@ -1,25 +1,65 @@
 import { readConfig } from "../../../config";
 import { logger } from "../../../logger";
+import type { CreateQueueBody } from "../../client";
 import { createQueue } from "../../client";
 import type {
 	CommonYargsArgv,
 	StrictYargsOptionsToInterface,
 } from "../../../yargs-types";
+import { UserError } from "../../../errors";
+import { handleFetchError } from "../../utils";
 
 export function options(yargs: CommonYargsArgv) {
-	return yargs.positional("name", {
-		type: "string",
-		demandOption: true,
-		description: "The name of the queue",
-	});
+	return yargs
+			.positional("name", {
+				type: "string",
+				demandOption: true,
+				description: "The name of the queue",
+			})
+		.options({
+			"delivery-delay": {
+				type: "number",
+				describe: "How long a published messages should be delayed for, in seconds. Must be a positive integer",
+			},
+			"no-delivery-delay": {
```

Comment:
I'm actually not sure why we need this option (for `create` Queue, at least).

Couldn't the user just omit the `delivery-delay` option?

(It might be needed for the `update` Queue command, when we add that)

Reply:
As discussed, at least for these, removed. Closes back a can of worms with the negated parameters at least for now.